config.yml.erb

```diff
@@ -264,6 +264,12 @@ google_client_secret:
 # Default AWS region for the application.
 aws_region: us-east-1
 
+# Override the endpoint used for S3, typically to point to an emulated local
+# instance of the service; LocalStack, MinIO, etc.
+aws_s3_endpoint:
+aws_s3_access_key_id:
+aws_s3_secret_access_key:
```

Comment:
I wouldn't mind this comment pointing to the docker/developers directory, I might find this first and then be wondering how to use it. The normal answer will probably be: try docker compose.

Reply:
I'd like to see this duplicated (along with the comment, and including a link to docker/developers) in locals.default.yml, I think that is a really useful file for "feature discovery".